src/App.tsx

```diff
@@ -1,61 +1,100 @@
+import { FC, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './api/todos';
+import { TodoList } from './components/TodoList';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [titileValue, setTitleValue] = useState<string>('');
+  const [selectedUser, setSelectedUser] = useState<number>(0);
+
+  const [isClicked, setIsClicked] = useState<boolean>(false);
+
+  const handleAdd = (event: React.FormEvent<HTMLFormElement>): void => {
+    event.preventDefault();
+
+    setIsClicked(true);
+
+    if (!titileValue.trim() || selectedUser === 0) {
+      return;
+    }
+
+    let newToDoId = 0;
+
+    todos.forEach((todo: Todo): void => {
+      if (todo.id > newToDoId) {
+        newToDoId = todo.id;
+      }
+    });
```

Comment:
The logic for determining the new todo ID is correct, but it could be simplified using `Math.max` and `map` to find the maximum ID more efficiently.